packages/nimble-components/src/rich-text-viewer/testing/rich-text-viewer.pageobject.ts

```diff
@@ -0,0 +1,27 @@
+import type { RichTextViewer } from '..';
+
+/**
+ * Page object for the `nimble-rich-text-viewer` component.
+ */
+export class RichTextViewerPageObject {
+    public constructor(
+        private readonly richTextViewerElement: RichTextViewer
+    ) {}
+
+    public getFirstChildElementTagName(
```

Comment:
Since you are passing in the `element` here, this function is responsible to just return the tag name of the element?

```suggestion
    public getTagName(
```

Reply:
If you meant to get first child element's tag name, then you can preserve this function and use `getFirstChildElement()` instead of passing the element as argument.

Reply:
Sure, I have updated the name to `getTagName()` as it is enough to get the text content when passing the `element` to it.